flepimop/gempyor_pkg/src/gempyor/model_info.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, config: confuse.ConfigView):
         self.tf = config["end_date"].as_date()
         if self.tf <= self.ti:
             raise ValueError(
-                "tf (time to finish) is less than or equal to ti (time to start)"
+                "`tf` (time to finish) is less than or equal to `ti` (time to start)."
```

Comment:
```suggestion
                f"Time to finish, '{tf}' is less than or equal to time to start, '{ti}'."
```
I think it's a bit redundant to include the variable name and the variable description. I just picked the description here, but fine with going the other way. Also helpful to include the dates.